.gitignore

```diff
@@ -22,4 +22,6 @@
 .idea/dataSources.xml
 .idea/sqlDataSources.xml
 .idea/dynamic.xml
-.idea/uiDesigner.xml
\ No newline at end of file
+.idea/uiDesigner.xml
+.idea/libraries/*
+
```

Comment:
please dont update gitignore here since we have not finished talking on how to handle .idea files